### PR TITLE
Clarifying and making error codes and messages consistent with API and App messages

### PR DESCRIPTION
**Pull request by gllmlbrt**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__VERSION__ = "0.5.1"
+__VERSION__ = "0.5.2"
```
Comment (astrandb) [inline Makefile:1]: Version bump should be handled in release flow. Multiple PR:s could be merged before releasing a new version.

File: custom_components/automower/__init__.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
     STATUS_EXECUTING_PARK,
     STATUS_EXECUTING_START,
     STATUS_EXECUTING_STOP,
+    STATUS_WAIT_POWER_UP,
```
Comment (astrandb) [inline custom_components/automower/__init__.py:29]: OK

File: custom_components/automower/const.py [hunk 3]
```diff
@@ -14,69 +14,140 @@
 DEFAULT_ICON = "mdi:robot-mower"
 VENDOR = "Husqvarna"
 VACUUM_SCAN_INTERVAL = 60
-VERSION = "0.5.1"
+VERSION = "0.5.2"
```
Comment (astrandb) [inline custom_components/automower/const.py:17]: Keep as is until release

File: custom_components/automower/const.py [hunk 4]
```diff
@@ -14,69 +14,140 @@
...
+STATUS_PARKED_AUTOTIMER =       'PARKED_AUTOTIMER'
+STATUS_PARKED_PARKED_SELECTED = 'PARKED_PARKED_SELECTED'
+STATUS_PAUSED =                 'PAUSED'
+STATUS_EXECUTING_PARK =         'EXECUTING_PARK'
+STATUS_EXECUTING_START =        'EXECUTING_START'
+STATUS_EXECUTING_STOP =         'EXECUTING_STOP'
+STATUS_WAIT_POWER_UP =          'WAIT_POWER_UP'
+STATUS_OFF_HATCH_OPEN =         'OFF_HATCH_OPEN'
+STATUS_OFF_HATCH_CLOSED =       'OFF_HATCH_CLOSED_DISABLED'
+STATUS_OFF_DISABLED =           'OFF_DISABLED'
```
Comment (astrandb) [inline custom_components/automower/const.py:36]: The code should be formatted with black, in the same way as the whole HA code-base. I agree that your suggestion is more legible, but black wins.

File: custom_components/automower/const.py [hunk 5]
```diff
@@ -14,69 +14,140 @@
...
+    STATUS_PARKED_AUTOTIMER:        { 'icon': 'mdi:weather-partly-cloudy',      'message': 'Parked (weather timer)' },
+    STATUS_PARKED_PARKED_SELECTED:  { 'icon': 'mdi:garage',          'message': 'Parked (manual)' },
+    STATUS_OK_SEARCHING:            { 'icon': 'mdi:magnify',        'message': 'Going to charging station' },
+    STATUS_EXECUTING_START:         { 'icon': 'mdi:dots-horizontal','message': 'Starting...' },
+    STATUS_EXECUTING_STOP:          { 'icon': 'mdi:dots-horizontal','message': 'Stopping...' },
+    STATUS_WAIT_POWER_UP:           { 'icon': 'mdi:dots-horizontal','message': 'Powering up...' },
+    STATUS_EXECUTING_PARK:          { 'icon': 'mdi:dots-horizontal','message': 'Preparing to park...' },
+    STATUS_OFF_HATCH_OPEN:          { 'icon': 'mdi:alert',          'message': 'Hatch opened' },
+    STATUS_OFF_HATCH_CLOSED:        { 'icon': 'mdi:pause',          'message': 'Stopped but not on base' },
+    STATUS_OFF_DISABLED:            { 'icon': 'mdi:close-circle-outline', 'message': 'Off'}
```
Comment (astrandb) [inline custom_components/automower/const.py:55]: The code should be formatted with black, in the same way as the whole HA code-base. I agree that your suggestion is more legible, but black wins.

File: custom_components/automower/manifest.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
 {
   "domain": "automower",
   "name": "Automower",
-  "version": "0.5.1",
+  "version": "0.5.2",
   "config_flow": true,
-  "documentation": "https://github.com/astrandb/automower_hass",
-  "issue_tracker": "https://github.com/astrandb/automower_hass/issues",
+  "documentation": "https://github.com/Nordicfastware/automower_hass",
+  "issue_tracker": "https://github.com/Nordicfastware/automower_hass/issues",
```
Comment (astrandb) [inline custom_components/automower/manifest.json:7]: I don't see the point. The users should be pointed to this repo. Of course you are free to use and maintain you own fork if you prefer.

**Review by astrandb — CHANGES_REQUESTED** [review]
Thank you for the PR. 
Added statuses and error messages are fine. However the formatting will be revoked by my release flow where flake8 and black are applied.
My general advice is to not put too much effort into improving this integration as this API is no longer supported by Husqvarna. E.g. they have recently stopped to send the mower model in the API response.

**Comment by gllmlbrt** [comment]
> Thank you for the PR.
> Added statuses and error messages are fine. However the formatting will be revoked by my release flow where flake8 and black are applied.
> My general advice is to not put too much effort into improving this integration as this API is no longer supported by Husqvarna. E.g. they have recently stopped to send the mower model in the API response.

Ok yes, I see that now, Thank you for your elaborate comments. 
I will withdraw and start over. And yes I agree no need to put too much effort into this. But the new API lacks the device tracker... :-(

You can reject the PR altogether. I will work more on my fork first...

**Pull request closed without merging**